include/irr/core/math/aabb2dSIMD.h

```diff
@@ -0,0 +1,169 @@
+#ifndef __IRR_AABB_SIMD_2D_H_INCLUDED__
+#define __IRR_AABB_SIMD_2D_H_INCLUDED__
+
+#include "irr/core/math/irrMath.h"
+#include "irr/core/memory/memory.h"
+#include "irr/core/alloc/AlignedBase.h"
+
+#include <iostream>
+
+#define SIGN_FLIP_MASK_XY	core::vector3du32_SIMD(0x80000000, 0x80000000, 0x00000000, 0x00000000)
```

Comment:
those need to be undefined at the end of the file to not pollute the preprocessor